### PR TITLE
DDF-4003 Update libraries to fix sonatype findings

### PR DESCRIPTION
**Pull request by kcover**
#### Link to 2.13.x port

https://github.com/codice/ddf/pull/3494

#### Who is reviewing it? 

@emmberk @peterhuffer @oconnormi 

#### Ask 2 committers to review/merge the PR and tag them here.

@clockard 
@vinamartin

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: features/pom.xml [hunk 1]
```diff
@@ -109,11 +109,11 @@
         </cxf.geronimo.transaction.version>
         <cxf.guava.version>${guava.version}</cxf.guava.version>
         <cxf.hazelcast.version>3.9.2</cxf.hazelcast.version>
-        <cxf.hibernate.validator.version>5.2.3.Final</cxf.hibernate.validator.version>
+        <cxf.hibernate.validator.version>5.3.6.Final</cxf.hibernate.validator.version>
         <cxf.httpcomponents.asyncclient.version>4.1.3</cxf.httpcomponents.asyncclient.version>
         <cxf.httpcomponents.core.version>4.4.9</cxf.httpcomponents.core.version>
         <cxf.james.mim4j.version>0.7.2</cxf.james.mim4j.version>
-        <cxf.jasypt.bundle.version>1.9.0_1</cxf.jasypt.bundle.version>
+        <cxf.jasypt.bundle.version>1.9.2_1</cxf.jasypt.bundle.version>
```
Comment (emmberk) [inline features/pom.xml]: ❗️ These changes should be documented in the comments above to keep record of what changes from the CXF defaults.

File: catalog/transformer/catalog-transformer-pdf/pom.xml [hunk 2]
```diff
@@ -25,7 +25,7 @@
     <packaging>bundle</packaging>
 
     <properties>
-        <pdfbox.version>2.0.2</pdfbox.version>
```
Comment (shaundmorris) [inline catalog/transformer/catalog-transformer-pdf/pom.xml]: ❓ should we have this property defined in one place, like the root pom.xml?

Comment (kcover) [reply catalog/transformer/catalog-transformer-pdf/pom.xml]: That's what I thought but I wasn't sure why we didn't do that in the first place. I'll try putting it in the root pom and see what happens.

**Comment by kcover** [comment]
build now

**Comment by clockard** [comment]
build now

**Pull request merged**